### DISCUSSION TITLE
Want to Dev list

### DISCUSSION THREAD

**Discussion by Thoooreau** [RetroAchievements/RAWeb#1868] [Ideas]
Basically the same as the Want to Play list, but exclusively for devs, where we can tag:

- games we want to create sets for in the future;
- games we want to revise in the future;
- maybe also specific achievements we want to change something about in the future?

It would be a good tool to help devs to better organize themselves.

**Comment by Thoooreau** [comment]
Ah, looks like it's already in the works in #1867.

**Reply by televandalist** [reply]
The first point, at least. Though I guess the second would probably work as well. The third may get a bit complicated 😆
